src/lib.rs

```diff
@@ -263,7 +263,11 @@ impl Frame {
         false
     }
 
-    fn print_source_if_avail(&self, mut out: impl WriteColor, s: &BacktracePrinter) -> IOResult {
+    pub fn print_source_if_avail(
```

Comment:
Hmm, why would you want to expose this one? This is an implementation detail that I'd prefer to keep private.

Reply:
Oh, sorry...I exposed it by mistake